recipes-connectivity/prplmesh/prplmesh/systemd-prplmesh.service

```diff
@@ -0,0 +1,19 @@
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Beerocks AP Manager service
+Requires=systemd-ubus.service
+After=systemd-wave_init.service
```

Comment:
Sounds like wave600-specific dependency. Such things probably should be added with `systemd-prplmesh.service.d/wave.conf` files from wave recipes.

Reply:
File has been deleted because not needed right now.
You comment was moved into issue #4